pkg/collector/python/memory.go

```diff
@@ -14,8 +14,10 @@ import (
 	"sync"
 	"unsafe"
 
+	"github.com/DataDog/datadog-agent/pkg/config"
 	"github.com/DataDog/datadog-agent/pkg/util/log"
 	_ "github.com/benesch/cgosymbolizer"
+	"github.com/cihub/seelog"
```

Comment:
is this import needed?

Reply:
It is, want to avoid casting the log level to a string, so using seelog native types. There was a typo when you reviewed, but it's fixed now.